.circleci/config.yml

```diff
@@ -22,293 +19,302 @@ parameters:
-
-      - persist_to_workspace:
-          name: "Upload zip file to shared workspace"
-          root: .
-          paths:
-            - << pipeline.parameters.function_zip_file >>
-
-  deploy:
-    description: Deploy application
+            az acr build
+            -t $TF_VAR_JMETER_DOCKER_IMAGE
+            -r $TF_VAR_JMETER_IMAGE_REGISTRY_NAME
+            -f docker/Dockerfile
+            .
+  
+  deploy_script:
     executor:
-      name: windows/default
+      name: azure-cli/default
     steps:
+      - checkout 
+
       - attach_workspace:
           at: .
 
+      - login-to-azure-down
+
       - run:
-          name: "Login to Azure"
+          name: "Invoking the deploy script for infrastructure and dsc configuration"
           command: |
-            # Login using service principal
-            az login `
-            --service-principal `
-            --tenant "$Env:TENANT_ID" `
-            --username "$Env:CLIENT_ID" `
-            --password "$Env:CLIENT_SECRET"
-
-            # Set default subscription
-            az account set `
-            --subscription "$Env:SUBSCRIPTION_ID"
+            cd scripts
+            chmod a+x env_setup.sh
+            ./env_setup.sh << pipeline.parameters.environment_name >>
 
       - run:
-          name: "Create azure function"
+          name: "Writing to properties file"
           command: |
-            # Set common variables which is associated with build id
-            $rg = "rg-jmetercircleci$Env:CIRCLE_BUILD_NUM"
-            $st = "stjmetercircleci$Env:CIRCLE_BUILD_NUM"
-            $func = "func-jmetercircleci$Env:CIRCLE_BUILD_NUM"
-
-            # Create resource group
-            az group create `
-            --location eastus `
-            --name "$rg"
-
-            # Create storage account
-            az storage account create `
-            --resource-group "$rg" `
-            --name "$st"
-
-            # Create azure function app
-            az functionapp create `
-            --resource-group "$rg" `
-            --storage-account "$st" `
-            --name "$func" `
-            --consumption-plan-location eastus `
-            --os-type Windows `
-            --runtime dotnet
-
-            # Deploy zip file
-            az functionapp deployment source config-zip `
-            --resource-group "$rg" `
-            --name "$func" `
-            --src << pipeline.parameters.function_zip_file >>
-
             # Output variables to file
-            echo "`$buildNum=`"$Env:CIRCLE_BUILD_NUM`"" >> output.ps1
-            echo "`$targetHost=`"$func.azurewebsites.net`"" >> output.ps1
+            # echo "TF_VAR_TARGET_HOSTNAME=$func.azurewebsites.net" >> .env
+            # echo "target_hostname=$func.azurewebsites.net" >> << pipeline.parameters.properties_file_name >>
+            echo "TF_VAR_TARGET_HOSTNAME=microsoft.com" >> .env
+            echo "target_hostname=microsoft.com" >> << pipeline.parameters.properties_file_name >>
```

Comment:
Are we going to update the hostname in the next integration?

Reply:
Yes